test/test_profile.py

```diff
@@ -22,7 +23,36 @@ def test_saving(default_params, task_manager_creator):
     manager, _ = task_manager_creator((manager_params, browser_params[:1]))
     manager.get(BASE_TEST_URL)
     manager.close()
-    assert (browser_params[0].profile_archive_dir / "profile.tar.gz").is_file()
+    tar_path = browser_params[0].profile_archive_dir / "profile.tar.gz"
+    assert tar_path.is_file()
+    # Test that the archived profile contains some basic items
+    profile_items = [
+        "cookies.sqlite",
+        "places.sqlite",
+        "webappsstore.sqlite",
+        "prefs.js",
+        "bookmarkbackups",
+        "cache2",
```

Comment:
I don't see this file in my profile but since the tests are passing it must be some quirk in my setup I guess

Reply:
Although I couldn't find this documented anywhere, after experimenting a little and reading the comments in https://stackoverflow.com/questions/25623705/selenium-retaining-firefox-cache-and-history-files, I concluded that when running Firefox with Selenium the browser cache is stored inside the profile, under the `cache2` directory, while when running Firefox normally it is stored under `~/.cache/mozilla/firefox/xxxxx/cache2`, where `xxxxx` is the same as the profile directory name.